gradle.properties

```diff
@@ -9,7 +9,7 @@ cordaProductVersion = 5.1.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 26
+cordaApiRevision = 27
```

Comment:
Is it a breaking change though?

I wonder if Avro will do defaulting for us automatically if the recipient has a new schema and payload does not have the value for the field?

FAO: @simon-johnson-r3

Reply:
Yes, I agree - looks like we will not be able o workaround without version bump.